docs/source/getting-started/installation.rst

```diff
@@ -92,6 +93,23 @@ Note: if you encounter *Authorization Error (Error 400: invalid_request)* with t
 
 Hint: run ``az account subscription list`` to get a list of subscription IDs under your account.
 
+**Lambda Cloud**
+
+Lambda Labs GPU Cloud is a cloud provider offering low-cost GPUs. You can learn more about them `here <https://lambdalabs.com/>`__.
+
+To configure Lambda Cloud access, go to `API Keys <https://cloud.lambdalabs.com/api-keys>`_ to generate a key and then add it to :code:`~/.lambda_cloud/lambda_keys` by running:
```

Comment:
Nit - make the location of the API Keys page clear for folks who can't see/access hyperlink
```suggestion
To configure Lambda Cloud access, go to `API Keys <https://cloud.lambdalabs.com/api-keys>`_ page on your Lambda Cloud console to generate a key and then add it to :code:`~/.lambda_cloud/lambda_keys` by running:
```

Reply:
Good idea.

Reply:
Updated!